### ISSUE TITLE
Truly open education will require sweeping changes

### ISSUE THREAD

**Issue by semioticrobotic** [open-organization-ambassadors/open-org-educators-guide#6]
Chapter proposal from Adam Haigler:

> Public education's aims—providing every student with a free and appropriate education regardless of race, sex, class, disability, etc.—are undeniably admirable. An unfortunate outgrowth of these aims is an emphasis on standardization that can be sluggish and stagnant. This wouldn't be such a problem if the world wasn't evolving so rapidly economically, technologically, politically, and socially. Parents, students, and employers are sounding the alarm about this issue. Clearly a structural shift will have to happen for the education system to become more responsive to the needs of its key stakeholders.

**Comment by engineerteacher** [comment]
Suggested questions:

1. What is the learning organization you work the closest with doing to adapt to the changes the author mentions - i.e. how is it ensuring that it is preparing students with the skills and dispositions needed to thrive in a rapidly changing world?
2. The author mentions three pillars of the Open Way Learning model. Describe the specific ways the learning organization you work the closest with is growing a culture of relentless collaboration, the free and open exchange of ideas & resources, and a systematic approach to innovation.
3. Going open can be risky as you put your work out there for others to review, critique, and adapt it to their own contexts. What steps have you or your organization taken to ensure people are comfortable with the vulnerability this open approach requires?

**Comment by trncb** [comment]
Some copyediting suggestions:

*1st paragraph:*
What are "the Three R's"?

*3rd paragraph:*
Not sure of the relevancy of the publish date? I propose the following;

From "Our thoughts are featured in our book, Open Up Education! How Open Way Learning Can Transform Schools, published in December 2018. We built on work from both David Price and ..."
To "Our thoughts are featured in our book, Open Up Education! How Open Way Learning Can Transform Schools, in which we built on work from both David Price and ..."

*4th paragraph:*
From "... lead excellent outcomes."
To ""... lead to excellent outcomes."

*After foundational principles*
I do not think this line is required - "Here's what that means."
It flows better to go straight into the explanation after stating the principles.

From "Apparently teachers are ..."
To "Apparently, teachers are ..."

From "... visiting each others' classrooms ..."
To "... visiting each other's classrooms ..."

*Pg 43, 2nd paragraph:*
From
"In the months since Open Up Education! was published and we launched our website, a flood of interest ..."
To
"Since publishing *Open Up Education!* and launching the *Open Way Learning* website, a flood of interest ..."

*Pg 45, 1st paragraph:*
From "... and OWL ambassadors ..."
To "... and Open Way Learning (OWL) ambassadors ..."

**Review questions**

*1st point:*
Remove double spaces "ensuring  that"

*3rd point:*
Remove double spaces "adapt  to"
